app/frontend/javascript/survey_widget.js

```diff
@@ -0,0 +1,88 @@
+// This is a fairly brute-force
+// attempt to make sure that if
+// you don't want to fill in the survey
+// or already did fill it in, you
+// won't be asked again for a long time.
+
+// Despite our best efforts,
+// the survey was not registering
+// the fact that users had dismissed it
+// or already filled it in.
+
+// Note that SurveySparrow has its own
+// mechanisms in place to attempt to ensure
+// that users only fill out the survey once;
+// we just don't trust them.
+jQuery( document ).ready(function() {
```

Comment:
Ideally we try to avoid using jQuery in code we write, becuase of a fantasy of eventually getting rid of it.

Not sure how long this code will end up staying around for... but sometimes even if we think short term, we end up keeping it for a while. Can we rewrite this without jQuery-isms? I don't think it uses many. Note we already have a `domready` thing that we use in other places to replace the `ready` itself. Then you just need to use built-into-browser function to find the `#ss_survey_widget` element, maybe that's it?

Reply:
Thanks for the `domready` tip - yeah, I'm optimistic I can get rid of all references to jQuery.

Reply:
Got rid of JQuery in this code yesterday.